metadata.py

```diff
@@ -0,0 +1,68 @@
+
+DO NOT CHANGE the name of the file
+"""
+
+from mmif import DocumentTypes, AnnotationTypes
+
+from clams.app import ClamsApp
+from clams.appmetadata import AppMetadata
+
+# DO NOT CHANGE the function name
+def appmetadata() -> AppMetadata:
+    """
+    Function to set app-metadata values and return it as an ``AppMetadata`` obj.
+    Read these documentations before changing the code below
+    - https://sdk.clams.ai/appmetadata.html metadata specification.
+    - https://sdk.clams.ai/autodoc/clams.appmetadata.html python API
+    
+    :return: AppMetadata object holding all necessary information.
+    """
+    
+    # first set up some basic information
+    metadata = AppMetadata(
+        name="Tfidf Keywordextractor",
+        description="extract keywords of a text document according to TF-IDF values. "
+                    "IDF values and all features come from related pickle files in the current directory."
+                    "App can either take a simple text document or take a MMIF file generated from the text slicer app.",  # briefly describe what the purpose and features of the app
+        app_license="Apache 2.0",  # short name for a software license like MIT, Apache2, GPL, etc.
+        identifier="tfidf-keywordextractor",  # should be a single string without whitespaces. If you don't intent to publish this app to the CLAMS app-directory, please use a full IRI format.
+        url="https://github.com/clamsproject/app-tfidf-keywordextractor",  # a website where the source code and full documentation of the app is hosted
+        # (if you are on the CLAMS team, this MUST be "https://github.com/clamsproject/app-tfidf-keywordextractor"
+        # (see ``.github/README.md`` file in this directory for the reason)
+        analyzer_version='', # use this IF THIS APP IS A WRAPPER of an existing computational analysis algorithm
+        # (it is very important to pinpoint the primary analyzer version for reproducibility)
+        # (for example, when the app's implementation uses ``torch``, it doesn't make the app a "torch-wrapper")
+        # (but, when the app doesn't implementaion any additional algorithms/model/architecture, but simply use API's of existing, for exmaple, OCR software, it is a wrapper)
+        # if the analyzer is a python app, and it's specified in the requirements.txt
+        # this trick can also be useful (replace ANALYZER_NAME with the pypi dist name)
+        # analyzer_version=[l.strip().rsplit('==')[-1] for l in open('requirements.txt').readlines() if re.match(r'^ANALYZER_NAME==', l)][0],
+        analyzer_license="",  # short name for a software license
+    )
+    # and then add I/O specifications: an app must have at least one input and one output
+    metadata.add_input(DocumentTypes.TextDocument)
+    metadata.add_output(DocumentTypes.TextDocument)
+    metadata.add_output(AnnotationTypes.Alignment)
+
+    metadata.add_parameter(name='idfFeatureFile',
```

Comment:
How about using `modelName` as the param name, as it's more general than "idf-feature-file" and should appeal to users who don't know what tfidf is.

Reply:
on the second though, how about comment this param out for now (since you only have one model file now) and leave some TODO comment for future revive?